helm/hpcc/templates/NOTES.txt

```diff
@@ -11,7 +11,11 @@ This chart has defined the following HPCC components:
 {{-      if hasKey $names $component.name -}}
 {{-       required ( printf "Component name %s in section %s was already seen in section %s" $component.name $sectionName (get $names $component.name)) nil -}}
 {{-      else }}
-  {{ $sectionName }}.{{ $component.name }}
+  {{ $sectionName }}.{{ $component.name }} 
```

Comment:
think there's a trailing whitespace here

Reply:
Fixed and repushed